docs/getting-started.md

```diff
@@ -106,13 +106,15 @@ latest=$(wget "http://spec.ccfc.min.s3.amazonaws.com/?prefix=firecracker-ci/v1.1
 wget "https://s3.amazonaws.com/spec.ccfc.min/${latest}"
 
 # Download a rootfs
-wget "https://s3.amazonaws.com/spec.ccfc.min/firecracker-ci/v1.10/${ARCH}/ubuntu-24.04.ext4"
-
-# Download the ssh key for the rootfs
-wget "https://s3.amazonaws.com/spec.ccfc.min/firecracker-ci/v1.10/${ARCH}/ubuntu-24.04.id_rsa"
-
-# Set user read permission on the ssh key
-chmod 400 ./ubuntu-24.04.id_rsa
+wget -O ubuntu-24.04.squashfs.orig "https://s3.amazonaws.com/spec.ccfc.min/firecracker-ci/v1.10/${ARCH}/ubuntu-24.04.squashfs"
```

Comment:
Nit: could we choose another name than an *.orig? Maybe it's just me, but I tend to use this suffix when creating custom files and backing up the originals.

Reply:
How about `upstream`?